ZCRMiOS/Classes/ZCRMDashBoardMetaComponent.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  ZCRMDashBoardMetaComponent.swift
+//  Pods
+//
+//  Created by Kalyani shiva on 30/07/18.
+//
+
+import Foundation
+
+public class ZCRMDashBoardMetaComponent {
```

Comment:
Shouldn't this be ZCRMDashBoardComponentMeta instead of ZCRMDashBoardMetaComponent ??

Reply:
Okay I will modify the name